app/mailers/user_email_mailer.rb

```diff
@@ -22,6 +22,6 @@ def response_invite(user, time_period)
   private
 
   def range_format(date)
-    "#{date.end_date.strftime('%d')} #{date.end_date.strftime('%b')}"
+    "#{date.due_date.strftime('%b')} #{date.due_date.strftime('%d')}"
```

Comment:
can be written like this 
def range_format(date)
  date.due_date.strftime('%b %d')
end

Reply:
Thanks Lyuba.